Common/Data/AffixRegistry.cs

```diff
@@ -103,8 +103,7 @@ select PoTMod.Instance.GetFileStream(path))
 	/// <returns>Instance of ItemAffix corresponding to the affix data.</returns>
 	internal static ItemAffix ConvertToItemAffix(ItemAffixData affixData)
 	{
-		string typeName = $"PathOfTerraria.Core.Systems.Affixes.ItemTypes.{affixData.AffixType}";
-
+		string typeName = $"PathOfTerraria.Common.Systems.Affixes.ItemTypes.{affixData.AffixType}";
```

Comment:
Just a minor tip but use ${PoTMod.ModName} when refering to the mod's name for content such as localization keys, etc. (might come in handy if there's ever an assembly rename, even if unlikely).